### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by composition-cassidy** [otomad/OtomadHelper#21] [Ideas]
![image](https://user-images.githubusercontent.com/99601533/213861833-fa2d7a0c-fd8a-40d9-9c9f-c5782eed4151.png)

So when exporting from FL Studio, most people usually place their percussion like this. 
When people use it on OTOMAD, all the perc visuals end up being Legato. Can you add a feature where you can set a limit on how long a visual can go?

**Comment by otomad** [comment]
It's a feature of FL Studio. You can re-import the midi file export from FL Studio that contains the percussion like this, you will find that it is actually done by FL Studio.
